### PR TITLE
Oct 2023 PSU config files for Microsoft

### PR DESCRIPTION
**Pull request by Luigi96**

### CODE REVIEW

File: linux/jdk/debian/src/main/packaging/microsoft/11/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+msopenjdk-11 (11.0.21-1) STABLE; urgency=medium
+
+  * Microsoft 11.0.21-1 release.
+
+ -- Microsoft Package Maintainers <openjdk@microsoft.com>  Wed, 11 Oct 2023 10:29:35 +0000
```
Comment (karianna) [inline linux/jdk/debian/src/main/packaging/microsoft/11/debian/changelog:5]: Is that date correct?

File: linux/jdk/debian/src/main/packaging/microsoft/21/debian/changelog [hunk 2]
```diff
@@ -1,5 +1,11 @@
+msopenjdk-21 (21.0.1-1) STABLE; urgency=medium
+
+  * Microsoft 21.0.1-1 release.
+
+ -- Microsoft Package Maintainers <openjdk@microsoft.com>  Thu, 12 Oct 2023 17:07:03 +0000
```
Comment (karianna) [inline linux/jdk/debian/src/main/packaging/microsoft/21/debian/changelog:5]: date check

File: linux/jdk/redhat/src/main/packaging/microsoft/11/msopenjdk-11.spec [hunk 3]
```diff
@@ -95,12 +93,12 @@ Provides: jre-11-%{java_provides}
 Provides: jre-%{java_provides}
 
 # First architecture (x64)
-Source0: %{source_url_base}/microsoft-jdk-%{upstream_version_url}-linux-%{vers_arch}.tar.gz
-Source1: %{source_url_base}/microsoft-jdk-%{upstream_version_url}-linux-%{vers_arch}.tar.gz.sha256sum.txt
+Source0: %{source_url_base}/microsoft-jdk-%{spec_version}-linux-%{vers_arch}.tar.gz
```
Comment (karianna) [inline linux/jdk/redhat/src/main/packaging/microsoft/11/msopenjdk-11.spec:96]: Is that change in version string sourcing correct?

Comment (Luigi96) [reply linux/jdk/redhat/src/main/packaging/microsoft/11/msopenjdk-11.spec:96]: Yes, I just double check it and it is working fine.

File: linux/jdk/debian/src/main/packaging/microsoft/21/debian/changelog (hunk 2 above)
Comment (Luigi96) [reply linux/jdk/debian/src/main/packaging/microsoft/21/debian/changelog:5]: I verified the date with the Team. This is the date when our pipelines build our latest package. This date is correct.

File: linux/jdk/debian/src/main/packaging/microsoft/11/debian/changelog (hunk 1 above)
Comment (Luigi96) [reply linux/jdk/debian/src/main/packaging/microsoft/11/debian/changelog:5]: I verified the date with the Team. The date is correct.

**Review by gdams — APPROVED** [review]

**Review by karianna — APPROVED** [review]

**Pull request merged**